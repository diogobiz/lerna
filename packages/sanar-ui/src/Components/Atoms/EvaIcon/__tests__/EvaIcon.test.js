import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESEvaIcon from '../'

it('renders correctly', () => {
    const component = <ESEvaIcon name='award-outline' />

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
