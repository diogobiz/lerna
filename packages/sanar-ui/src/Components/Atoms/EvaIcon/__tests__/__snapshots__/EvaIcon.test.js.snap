// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<i
  className="anticon es-eva-icon es-eva-icon--medium"
>
  <i
    data-eva="award-outline"
    data-eva-fill="#fff"
    data-eva-height="18px"
    data-eva-width="18px"
  />
</i>
`;
